src/Auth.js

```diff
@@ -1,5 +1,13 @@
-const RestQuery = require('./RestQuery');
```

Comment:
What's the reason behind removing this and then `require`ing it in line 3 other times?

Reply:
Circular dependency. Auth.js is used into RestQuery. So the RestQuery module should be loaded dynamically when needed.